### ISSUE TITLE
复现

### ISSUE THREAD

**Issue by HITPaperPlane** [haonan3/SilentBadDiffusion#1]
真的有人复现成功了么...

**Comment by haonan3** [comment]
Hi there,

Thanks for reaching out. Could you let me know which specific issue you’re running into? I’d be happy to help!

**Comment by haonan3** [comment]
Hi bro, sorry Maybe I cannot promptly reply because I am traveling. I tested the cleaned version before separating it from my exp code. But the dependent libraries and repos might have changed over time. If you need help and discussion immediately, feel free to find my contact on WeChat or Twitter. We can have a quick online meeting~       :  )

**Comment by HITPaperPlane** [comment]
感谢您的回复，请问您能否提供一下poisoning_data_generation.py 和 target_model_training.py 运行时的具体参数配置，想知道您是对哪张图片做的投毒和微调以达到后门攻击，您的代码我运行成功了，就是微调之后生成的图片没有到达0.5以上的相似度



---- 回复的原邮件 ----
| 发件人 | Haonan ***@***.***> |
| 发送日期 | 2024年12月8日 09:14 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [haonan3/ICML-2024-Oral-SilentBadDiffusion] 复现 (Issue #1) |

Hi there,

Thanks for reaching out. Could you let me know which specific issue you’re running into? I’d be happy to help!

**Comment by haonan3** [comment]
Emmmm, may I ask which dataset you are working with? Maybe we can check the poisoning data generated from your side and make sure it is correct. For me, I just sampled some images (I remember 20 images) from the two sources.

**Comment by haonan3** [comment]
For the specific images, not sure, but let me try my best to find it for you. ( For Pokémon data, I am sure the image of demo example is used. Maybe you can take it as a starting point.

**Comment by HITPaperPlane** [comment]
感谢您的回复
我使用的是Pokémon数据集，投毒时测试了0号到50号的图片，有些特征即使重复次数修补模型也无法生成sscd值小于0.5的图片（可能是gpt-4的问题），因此我放弃了这些特征（这可能是我没有能完全复现的原因）。我这边只需要几个Pokémon图片的例子能使我复现攻击即可（最好能提供相应的特征提取结果），我就可以完成接下来的实验。（另外我想请问投毒数据的数量是否有关系，我在论文中只看到了比例）。

---- 回复的原邮件 ----
| 发件人 | Haonan ***@***.***> |
| 发送日期 | 2024年12月8日 16:25 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [haonan3/ICML-2024-Oral-SilentBadDiffusion] 复现 (Issue #1) |

Emmmm, may I ask which dataset you are working with? Maybe we can check the poisoning data generated from your side and make sure it is correct. For me, I just sampled some images (I remember 20 images) from the two sources.

**Comment by haonan3** [comment]
For why we should check the generated data:

The success actually depends on GPT, and its backbone model will change over time. Therefore, we should examine the data to ensure it’s quality.

**Comment by haonan3** [comment]
Yes. I think the total amount of data is also important. Yes🫡Let me find the data for you. Please also check the data generated from your side. 

Why I think it is weird: I remembered the process is smooth especially on the Pokémon dataset. I do spend some time on the midjourney , but the Pokémon is not that hard.

**Comment by HITPaperPlane** [comment]
感谢您的回复，这是对于11号图片的一些投毒图片的例子，gpt将特征分解为a nose in front of its mouth a tail at back three legs below its body two eyes on its face，最后微调后生成的best image是这样，我猜想可能是gpt提取出的特征不太明显。按照您的建议，我接下来打算尝试您论文中的示例图片并继续检查我的过程（我这边投毒数据的数量设置的是50），很抱歉占用您的旅行时间，再次感谢您的回复。


---- 回复的原邮件 ----
| 发件人 | Haonan ***@***.***> |
| 发送日期 | 2024年12月8日 16:48 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [haonan3/ICML-2024-Oral-SilentBadDiffusion] 复现 (Issue #1) |

Yes. I think the total amount of data is also important. Yes🫡Let me find the data for you. Please also check the data generated from your side.

Why I think it is weird: I remembered the process is smooth especially on the Pokémon dataset. I do spend some time on the midjourney , but the Pokémon is not that hard.

**Comment by haonan3** [comment]
You are welcome! Bro. 

(Sry. I can only reply to you from my iPhone now)
